src/pudl_archiver/cli.py

```diff
@@ -77,6 +81,7 @@ def parse_main(args=None):
     parser.add_argument(
         "--depositor",
         type=str,
+        help="Specifies what backend engine will be used for archive storage. Current allowable options include zenodo and fsspec",
```

Comment:
Last nit: We could use the `choices` parameter here to restrict to these valid options? https://docs.python.org/3/library/argparse.html#choices

Reply:
Good call, added!